PlayCover/Views/App Views/StoreAppView.swift

```diff
@@ -7,6 +7,57 @@
 
 import SwiftUI
 
+struct ITunesResult: Decodable {
+    var isGameCenterEnabled: Bool
+    var supportedDevices: [String]
+    var features: [String]
+    var advisories: [String]
+    var screenshotUrls: [String]
+    var ipadScreenshotUrls: [String]
+    var appletvScreenshotUrls: [String]
+    var artworkUrl60: String
+    var artworkUrl512: String
+    var artworkUrl100: String
+    var artistViewUrl: String
+    var kind: String
+    var isVppDeviceBasedLicensingEnabled: Bool
+    var currentVersionReleaseDate: String
+    var releaseNotes: String
+    var description: String
+    var trackId: Int
+    var trackName: String
+    var bundleId: String
+    var sellerName: String
+    var genreIds: [String]
+    var primaryGenreName: String
+    var primaryGenreId: Int
+    var currency: String
+    var formattedPrice: String
+    var contentAdvisoryRating: String
+    var averageUserRatingForCurrentVersion: Float
+    var userRatingCountForCurrentVersion: Int
+    var trackViewUrl: String
+    var trackContentRating: String
+    var averageUserRating: Float
+    var minimumOsVersion: String
+    var trackCensoredName: String
+    var languageCodesISO2A: [String]
+    var fileSizeBytes: String
+    var releaseDate: String
+    var artistId: Int
+    var artistName: String
+    var genres: [String]
+    var price: Float
+    var version: String
+    var wrapperType: String
+    var userRatingCount: Int
```

Comment:
Do we need all of these? Comment out unused ones.

Reply:
This data may be useful in other places in the UI. I'll extract it to its own file for easier readability.